### PR TITLE
Most Popular Products docs page

### PR DESCRIPTION
**Pull request by borisrev**
Creation section and page

# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Currents<br>Internal Tools<br>Product Partnerships<br>SMS<br>Customer Lifecycle, Identity and Permissions</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Reporting<br>Intelligence<br>User Targeting<br>IAM<br>Channels<br>FIX</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging and Automation<br>Email (Composition and Infrastructure)</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/mostpopular.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+alias: "/catalogs/"
+page_order: 13
+description: "This reference article covers how to create a Recommendation for Most Popular Products."
+---
+
+# Most Popular Produts
+
+{% alert important %} 
+Most Popular Products is currently in early access. Contact your Braze customer success manager if you're interested in participating. 
+{% alert important %}
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/mostpopular.md:13]:
```suggestion
{% endalert %} 
```

File: _docs/_user_guide/personalization_and_dynamic_content/catalogs/mostpopular.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+In the "Type" dropdown, select "Most popular products".
+
+### Product IDs
+
+The final step is to tell Braze where to find the product IDs in the purchase events. In order to create this or any other Recommendation, the unique values in the `id` field of the Catalog must also be in the `product_id`  field of the purchase events or in the top level fields of the `properties` of those evengts.  The dropdown labeled Product IDs on the creation page will be prepopulated with those fields sent via SDK to Braze. Select the one that corresponds to the product IDs.
+
+{% alert important %} 
+The product IDs must be in a top level field of the properties of the Purchase event. No nested properties or custom event fields are supported at this time.
+{% alert important %}
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/personalization_and_dynamic_content/catalogs/mostpopular.md:42]:
```suggestion
{% endalert %} 
```

**Review by mmwillet — APPROVED** [review]
This is fine; though I suspect that we may want to include some more tangible examples of how to use the templating and describe the creation flow a little more.
1. for examples of templating I think displaying how to access and display / use fields will be helpful even something akin to this simple example:
```
{% assign recs = product_recommendation.${REC_NAME} %}
{{ recs[0].CATALOG_COLUMN_A }}
{{ recs[0].CATALOG_COLUMN_B }}
```
might be helpful
2. The key thing is that we need to note that clients needs to wait about 5 minutes for recs to populate before they can be used in a message.

**Comment by bre-fitzgerald** [comment]
@borisrev made some changes to clean up and fix the flow, as well as move to the hidden section. Long term I think this makes more sense to keep under the **Predictive Suite** section. When we add more recommendation types, we can transition the **Item Recommendations** page to be a landing page, and nest recommendation-type articles underneath (or use tabs/sections for different recommendation types, depending on the content needs).

When this GAs, I'll add a cross-reference in the catalogs docs to link out to this page where appropriate. This makes more sense flow-wise than including it in the catalogs section, because the feature itself lives under predictions. LMK if you have any questions/feedback on the revisions I made. 🖖

**Comment by borisrev** [comment]
@mmwillet  

1. can we link to other resources to explain liquid usage? or is it unusual for this particular feature ?
2. will add a note about the time delay.

**Review by bre-fitzgerald — APPROVED** [review]

**Comment by bre-fitzgerald** [comment]
@cla-bot check

**Pull request merged**